torchtitan/experiments/qwen3/train_configs/qwen3_0.6b.toml

```diff
@@ -0,0 +1,61 @@
+# torchtitan Config.toml
+# This is addapted from the original config file in the llama repo
+
+[job]
+dump_folder = "./outputs"
+description = "Qwen 3 0.6B training"
+
+[profiling]
+enable_profiling = true
+save_traces_folder = "profile_trace"
+profile_freq = 100
+
+[metrics]
+log_freq = 10
+enable_tensorboard = true
+save_tb_folder = "tb"
+
+[model]
+name = "qwen3"
+flavor = "0.6B"
+tokenizer_path = "./assets/tokenizer/Qwen3-0.6B"
+# converters = ["float8"]
+
+[optimizer]
+name = "AdamW"
+lr = 3e-4
```

Comment:
Please double check if `lr`, and lr_schedule reasonable  -  We want user can directly start training using this set of parameter.